warehouse/deltalake/deltalake.go

```diff
@@ -127,6 +132,14 @@ func loadConfig() {
 	config.RegisterStringConfigVariable("MERGE", &loadTableStrategy, true, "Warehouse.deltalake.loadTableStrategy")
 }
 
+func loadExcludeColumns() {
+	// Since event_date is an auto generated column in order to support partitioning.
+	// We need to ignore it during query generation.
+	excludeColumnsMap = map[string]bool{
```

Comment:
we don't we declare this map at top for once
var excludeColumnsMap = map[string]bool{}

Reply:
Yes. Moving it now.